Dockerfile

```diff
@@ -68,12 +57,13 @@ COPY requirements/python.txt /tmp/requirements/python.txt
 RUN PIP_INSTALL="python -m pip --no-cache-dir install --upgrade" && \
     $PIP_INSTALL pip && \
     $PIP_INSTALL -r /tmp/requirements/python.txt -f https://download.pytorch.org/whl/torch_stable.html && \
-    rm /tmp/requirements/python.txt && \
+    rm -r /tmp/requirements && \
 # ==================================================================
 # VSCode server
 # ------------------------------------------------------------------
-    wget https://github.com/cdr/code-server/releases/download/v3.9.1/code-server_3.9.1_amd64.deb  && \
-    dpkg -i code-server_3.9.1_amd64.deb
+    wget -q https://github.com/cdr/code-server/releases/download/v3.9.1/code-server_3.9.1_amd64.deb  && \
```

Comment:
Since you're updating the code, let's use `v3.11.1`

Reply:
done in 
5c410df